.bundle/config

```diff
@@ -1,2 +0,0 @@
----
-BUNDLE_FROZEN: "1"
```

Comment:
Please remove this change, and use `bundle exec rake dependencies:update_gemfile_lock` to ensure the Gemfile.lock is correct.